test/server/spec.js

```diff
@@ -171,6 +171,22 @@ describe('superagentCache', function(){
       );
     });
 
+    it('.get() .pruneKey() .end() should query with all params but create a key without the indicated params', function (done) {
+      superagent
+        .get('localhost:3007/one')
+        .use(superagentCache)
+        .pruneKey(function (key) {
+          key.uri = 'localhost:3007/false';
+          return key;
+        })
+        .end(function (err, response, key){
+          expect(key.indexOf('/one')).toBe(-1);
+          expect(key.indexOf('/false')).toBeGreaterThan(-1);
```

Comment:
nit: the `.toContain` matcher would be a better choice if it's available.

Reply:
Yeah, unfortunately that's what is used in the other tests - I don't think I would've written the tests this way...

Reply:
Thanks for following their coding style even though it's...idiosyncratic.